src/detect-flowbits.c

```diff
@@ -704,17 +716,96 @@ int DetectFlowbitsAnalyze(DetectEngineCtx *de_ctx)
         DetectFlowbitsAnalyzeDump(de_ctx, array, array_size);
     }
 
-end:
+    FBAnalyzerFree(&fba);
+    return 0;
+error:
+    FBAnalyzerFree(&fba);
+    return -1;
+}
+
+// TODO misses IPOnly rules. IPOnly flowbit rules are set only though.
+static struct FBAnalyzer DetectFlowbitsAnalyzeForGroup(
```

Comment:
Do we need to document or highlight this anywhere in the userguide?

Reply:
Is this mostly user transparent?